x11-plugins/gkwebmon/gkwebmon-0.2-r3.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gkrellm-plugin toolchain-funcs
+
+DESCRIPTION="A web monitor plugin for GKrellM2"
+HOMEPAGE="http://gkwebmon.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tgz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~sparc ~x86"
+
+# The Makefile links with -lssl.
+RDEPEND="
+	app-admin/gkrellm:2[X]
+	dev-libs/openssl:0="
```

Comment:
nit: just drop the 0, openssl hasn't been slotted for a long time (think there was a failed experiment for a bit)

Reply:
Done.